build-system/eslint-rules/query-selector.js

```diff
@@ -157,26 +177,35 @@ module.exports = function (context) {
         });
       }
 
-      selector = quasis.join('');
+      selector = accumulator + quasis[quasis.length - 1];
```

Comment:
was this causing issues?

Reply:
Without this, it was ignoring the final static portion of the template literal. Eg, `foo${expression}bar` would leave out `bar`.